exp-player/addon/components/exp-rating-form/component.js

```diff
@@ -378,13 +380,14 @@ var questions = [
                 scale: SEVEN_POINT_SCALE,
                 value: null,
                 labelTop: false,
+                formatLabel: 'measure-six',
                 labels: [
                     {
                         rating: 0,
```

Comment:
We want the labels to say 1-7... Should the internal values we map to that scale match? (setting a range 0-6 seems a bit odd)